COMET.Web.Common/ViewModels/Components/OpenModelViewModel.cs

```diff
@@ -222,6 +222,11 @@ public virtual async Task<Result<Iteration>> OpenSession()
                 return Result.Fail(["The selected iteration and the domain of expertise should not be null"]);
             }
 
+            if (this.sessionService.OpenIterations.Items.Any(x => x.IterationSetup.Iid == this.SelectedIterationSetup.IterationSetupId))
+            {
+                return Result.Fail(["The selected iteration is already openened"]);

```

Comment:
what happens for the user? if the user should not be able to select an iteration that is already open, lets prevent them from doing so. The feedback message can stay.